ghost/core/test/unit/frontend/helpers/ghost_head.test.js

```diff
@@ -375,6 +440,57 @@ describe('{{ghost_head}} helper', function () {
             }).catch(done);
         });
 
+        it('does not inject count script if comments off', function (done) {
+            settingsCache.get.withArgs('comments_enabled').returns('off');
+
+            testGhostHead(testUtils.createHbsResponse({
+                locals: {
+                    relativeUrl: '/',
+                    context: ['home', 'index'],
+                    safeVersion: '0.3'
+                }
+            })).then(function (rendered) {
+                should.exist(rendered);
+                should(rendered.string).not.containEql('<script defer src="/public/comment-counts.min.js?v=asset-hash" data-ghost-comments-counts-api="http://localhost:65530/members/api/comments/counts/"></script>');
```

Comment:
In this case it was very hard to see if the actual snapshot files were right (5K+ added lines) hence the reason I added explicit checks, but indeed they can get removed know, because changes should become visible in future PR's and test runs. I'll remove the explicit checks here, and will take a look at the other tests and remove these checks where possible (but maybe leave it for some more complicated situations where you cannot know the output without knowing the original context).